pipelines/README.md

```diff
@@ -0,0 +1,15 @@
+# Pipeline Definitions
+
+This directory contains pipeline definition YAML files which are used when processing data with the LSST Science Pipelines.
+
+The pipelines defined here come in three flavors: camera-specific (within named directories), camera-agnostic (top-level, if any), and building-block ingredients (within the [\_ingredients](_ingredients) directory).
+Pipelines within the ingredients directory tend to be imported by other pipelines, and are not intended to be used directly by end-users for science purposes.
```

Comment:
Nothing in `ap_verify` should be used for science purposes, something I occasionally butt heads with would-be users about. Perhaps: 
```suggestion
Pipelines within the ingredients directory tend to be imported by other pipelines, and are not intended to be used directly by end-users for verification purposes.
```

Reply:
This may be a matter of definition - isn't verification a part of the scientific process? If this is a point of contention then we could just remove the latter part of this sentence, ending with "by end-users."

Reply:
When I hear a phrase like "science runs" or "scientific processing", it usually means running a pipeline on a generic set of data. The `ap_verify` pipelines, unlike the `ap_pipe` pipelines, are not designed to be run on anything other than a packaged `ap_verify` dataset -- it might work, but it's very Not Supported.

I'm fine with just removing the final phrase.